.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: java
+jdk:
+  - oraclejdk8
+script:
+  - gradle build
```

Comment:
Is it possible to use `./gradlew` instead of the built-in `gradle` installation coming from the build server?

Reply:
That throwed me an error in my first try, but I'm going to try again because is possible I didn't put the ./ at the beginning.